api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -228,15 +261,21 @@ public void findCriteriaBy() {
         .isEqualTo(
             controller
                 .findCriteriaBy(
-                    1L, Domain.CONDITION.toString(), CriteriaType.ICD9CM.toString(), false, 0L)
+                    WORKSPACE_NAMESPACE,
+                    WORKSPACE_ID,
+                    Domain.CONDITION.toString(),
+                    CriteriaType.ICD9CM.toString(),
+                    false,
+                    0L)
                 .getBody()
                 .getItems()
                 .get(0));
```

Comment:
the assert is backwards in the test. We should assert that the controller call is equal to the createResponseCriteria. Seems that this exists in alot of tests in this class.